docs/_data/metrics/social_media/facebook_weekly.csv

```diff
@@ -0,0 +1,33 @@
+week_starting,audience_last,engagements_total,audience_gained_total,impressions_total
```

Comment:
Move individual social media files into docs/_data/metrics/social_media/service/facebook_weekly.csv (same for twitter, linkedint and instagram)